### PR TITLE
feat: collect version metric

### PR DESCRIPTION
**Pull request by alexanbj**
Similar to https://github.com/podium-lib/podlet/pull/24

### CODE REVIEW

File: lib/layout.js [hunk 1]
```diff
@@ -113,6 +114,27 @@ const PodiumLayout = class PodiumLayout {
...
         });
 
+        // Skip a tick to ensure the metric stream has been consumed
+        setImmediate(() => {
+            const moduleVersion = pkg.version;
+            const segments = moduleVersion.split('.').map(value => {
+                return parseInt(value, 10);
+            });
+
+            this.metrics.metric({
```
Comment (digitalsadhu) [inline lib/layout.js]: in keeping with new metrics syntax, this should really be:

```js
// defined somewhere
const versionGauge = this.metrics.gauge({
    name: 'podium_layout_version_info',
    description: '@podium/layout version info',
);

// used when collecting the metric
versionGauge.set(1, {
    labels: { 
        version: moduleVersion,
        major: segments[0],
        minor: segments[1],
        patch: segments[2],
    }
});
```

File: lib/layout.js [hunk 2]
```diff
@@ -113,6 +114,27 @@ const PodiumLayout = class PodiumLayout {
             value: new Metrics(),
         });
 
+        // Skip a tick to ensure the metric stream has been consumed
+        setImmediate(() => {
```
Comment (digitalsadhu) [inline lib/layout.js]: what happens when you don't do this?

Comment (alexanbj) [reply lib/layout.js]: The metric is lost we are inside the constructor

File: lib/layout.js (hunk 1 above)
Comment (trygve-lie) [reply lib/layout.js]: +1 on this.

**Pull request merged**